src/RedisOptions.php

```diff
@@ -261,4 +260,26 @@ public function getPassword()
     {
         return $this->getResourceManager()->getPassword($this->getResourceId());
     }
+
+    /**
+     * Set resource user
+     *
+     * @param string $user ACL User
+     * @return RedisOptions Provides a fluent interface
+     */
+    public function setUser($user)
+    {
+        $this->getResourceManager()->setUser($this->getResourceId(), $user);
+        return $this;
+    }
+
+    /**
+     * Get resource user
+     *
+     * @return string
+     */
+    public function getUser()
```

Comment:
Is always a `string` returned, even if nothing was set beforehand?
If so, please add native return type.
If `null` is returned if user was not previously configured via `setUser`, the return value should reflect all possible return values.